build_projects/dotnet-cli-build/project.json

```diff
@@ -13,15 +13,27 @@
     "System.Xml.XmlSerializer": "4.0.11-rc2-24027",
     "WindowsAzure.Storage": "6.2.2-preview",
 
-    "Microsoft.DotNet.Cli.Build.Framework": {"target":"project"},
+    "Microsoft.DotNet.Cli.Build.Framework": "1.0.0-*",
```

Comment:
I think this should stay {target: project}.  From what I've heard that is the correct way to P2P reference.